setup.py

```diff
@@ -10,26 +10,26 @@
 except ImportError:
     cythonize = False
 
-# defines whether the python interface should link againt a static (.a)
-# or a shared (.so) scipopt library
-usesharedlib = True
-pathToScipoptsuite = os.path.abspath('../../../')
-if usesharedlib:
-    libscipopt = 'lib/libscipopt.so'
-else:
-    libscipopt = 'lib/libscipopt.a'
-
-linkincludescip = 'lib/scip-src'
+pathToSCIPOpt = os.path.abspath('../../../')
+pathToSCIPsrc = os.path.abspath('../../src')
```

Comment:
We still want to publish the interface together with a regular SCIP
release, so these relative paths are still valid for this case.

On Wed, Jun 29, 2016, 19:03 Robert Schwarz <notifications@github.com> wrote:

> In setup.py
> <https://github.com/SCIP-Interfaces/PySCIPOpt/pull/11#discussion_r68986384>
> :
>
> > @@ -10,26 +10,26 @@
> >  except ImportError:
> >      cythonize = False
> >
> > -# defines whether the python interface should link againt a static (.a)
> > -# or a shared (.so) scipopt library
> > -usesharedlib = True
> > -pathToScipoptsuite = os.path.abspath('../../../')
> > -if usesharedlib:
> > -    libscipopt = 'lib/libscipopt.so'
> > -else:
> > -    libscipopt = 'lib/libscipopt.a'
> > -
> > -linkincludescip = 'lib/scip-src'
> > +pathToSCIPOpt = os.path.abspath('../../../')
> > +pathToSCIPsrc = os.path.abspath('../../src')
>
> Do these relative paths still make sense, now that PySCIPOpt is no longer
> in the scip repository?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SCIP-Interfaces/PySCIPOpt/pull/11/files/6511e46d3e2888d057c823e4f464bd838d078aea#r68986384>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABb-s9OuAK7OLmlnoWl0XcURpgm7N4ruks5qQqVUgaJpZM4JBVv->
> .
>